src/user/redux/modules/change_password.js

```diff
@@ -1,12 +1,20 @@
-import { FETCHED_MEMBER } from '../../../shared/redux/modules/member.js'
+import { createAction } from 'redux-actions'
```

Comment:
It's like tests. It makes sure everything glues together correctly. If you use it a bit more widely than we do, you can get a fair bit of confidence that when you make a change to a function signature in one place, you'll have correctly changed all the places that consume it